### PR TITLE
Feature/81: Performance Editor UI

### PR DESCRIPTION
**Pull request by ppator1**

### CODE REVIEW

File: BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+        }
+
+        /**
+         * User has submitted PerformanceEquationEditorDialog result
+         * @param equation The submitted equation string
+         */
+        onSubmitEquationEditorResult(equation: string) {
+            // reset equationEditorDialogData
+            this.equationEditorDialogData = {...emptyEquationEditorDialogData};
+            if (!isNil(equation)) {
```
Comment (saranshbhardwaj) [inline BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue]: can't we use `hasvalue` over here?

File: BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+        }
+
+        /**
+         * User has submitted CriteriaEditorDialog result
+         * @param criteria The submitted criteria string
+         */
+        onSubmitCriteriaEditorDialogResult(criteria: string) {
+            // reset criteriaEditorDialogData
+            this.criteriaEditorDialogData = {...emptyCriteriaEditorDialogData};
+            if (!isNil(criteria)) {
```
Comment (saranshbhardwaj) [inline BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue]: can we use `hasvalue` here ?

File: BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue (hunk 1 above)
Comment (ppator1) [reply BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue]: We can, but then that would disallow users from actually removing a criteria (an empty string). I have a feeling they may want to be able to do that.

Comment (saranshbhardwaj) [reply BridgeCareApp/VuejsApp/src/components/performance-editor/PerformanceEditor.vue]: got it.

**Pull request merged**